### PR TITLE
Recent works and about me section

### PR DESCRIPTION
**Pull request by calvnce**
The update adds the recent works and the about me section

### CODE REVIEW

**Comment by calvnce** [comment]
You may notice some differences on the image sizes, the difference is caused by the snipping too (Snipping accuracy)

File: public/index.html [hunk 1]
```diff
@@ -59,6 +59,175 @@ <h1>
...
+    <!-- Current Works Section -->
+    <section class="works-container">
+      <div class="header">
+        <h1>My Recent <br/>Works</h1>
+        <div id="underline"></div>
+      </div>
+      <!-- Cads 
+       1st Card
+      -->
+      <div class="card">
```
Comment (smcommits) [inline public/index.html:75]: - [ ] **Please use `ul` tag to group all the projects and `li` tag for each project.**

What this means is that you should use the `ul` tag to group all the projects together in one element. Then, inside of that `ul` element, you should use the `li` tag to display each project and its contents.

Why is it important?

Whenever you are trying to group and list things, you should consider using `ul` and `li` tags. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.

File: public/index.html [hunk 2]
```diff
@@ -59,6 +59,175 @@ <h1>
...
+        <div class="project-image"></div>
+          <div class="project-info">
+            <div class="project-name">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+          <div class="skills">
```
Comment (smcommits) [inline public/index.html]: - [ ] **Please `ul` tag to group all the technologies and `li` tag for each technology**

Again, as mentioned above, whenever you are trying to list things, `ul` and `li` tags will help you to achieve better accessibly and optimization.

File: public/index.html [hunk 3]
```diff
@@ -59,6 +59,175 @@ <h1>
...
+      <!-- Frameworks -->
+      <div class="card simple-card">
+        <div class="image">
+          <img src="./img/icon-frameworks.png" alt="Framworks">
+        </div>
+        <div class="outline-wrapper">
+          <div class="details">
+            <h2>Framworks</h2>
+          </div>
+          <div class="items">
```
Comment (smcommits) [inline public/index.html]: - [ ] **Please use `ul` tags for grouping all the skills and `li` tags for each skill**

Again, for the same reason, as mentioned above. 😄

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @calvnce , 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by calvnce** [comment]
The changes suggested on the use of `ul` `li` tags for listing is successfully covered

**Pull request merged**